game/game_application.cpp

```diff
@@ -14,6 +14,21 @@
 
 #include "systems/game_render_system.h"
 
+bool checkReservedName(std::string sceneName)
```

Comment:
Rename this to `isReservedName`
Declare this function only once, in scene.h and define it in scene.cpp

Reply:
done